src/shared/nt-results/ResultsByLoad.react.js

```diff
@@ -0,0 +1,194 @@
+import * as d3 from 'd3'
+import gql from 'graphql-tag'
+import _ from 'lodash'
```

Comment:
I added UglifyJs to simplify the bundle size, but it could be done better in the future in order to optimize.